apis/v1alpha1/httproute_types.go

```diff
@@ -233,7 +233,7 @@ const (
 	// header from an HTTP request before it is sent to the upstream target.
 	// Support: core
 	// +optional
-	FilterHTTPRequesttHeader = "RequestHeader"
+	FilterTypeHTTPRequestHeader = "RequestHeader"
```

Comment:
Nit: I think other filter type vars should also be prefixed with FilterType as part of this change.

Reply:
Another nit, it looks like the comment above this has not been updated to reflect the new var name.

Reply:
Apologies for the dumb mistakes, PTAL.